### PR TITLE
Add json files and code for emitting event for google sign in.

### PR DESCRIPTION
**Pull request by danialmalik**
**Description:** _Emit event when user signs in using google authentication_

**Checks before merge:**

- [ ] Reviewed
- [ ] Commits squashed

### CODE REVIEW

File: common/djangoapps/third_party_auth/pipeline.py [hunk 1]
```diff
@@ -676,6 +676,23 @@ def login_analytics(strategy, auth_entry, current_partial=None, *args, **kwargs)
         )
 
 
+def emit_login_event(strategy, auth_entry, current_partial=None, *args, **kwargs):
```
Comment (asamolion) [inline common/djangoapps/third_party_auth/pipeline.py]: Please move this function to our own caliper app.

File: common/djangoapps/third_party_auth/settings.py [hunk 2]
```diff
@@ -61,6 +61,7 @@ def apply_settings(django_settings):
         'third_party_auth.pipeline.set_id_verification_status',
         'third_party_auth.pipeline.set_logged_in_cookies',
         'third_party_auth.pipeline.login_analytics',
+        'third_party_auth.pipeline.emit_login_event',
```
Comment (asamolion) [inline common/djangoapps/third_party_auth/settings.py]: I believe you may be able to update `SOCIAL_AUTH_PIPELINE` setting by configuration. In that case we wouldn't need to modify the code.

File: common/djangoapps/third_party_auth/pipeline.py (hunk 1 above)
Comment (danialmalik) [reply common/djangoapps/third_party_auth/pipeline.py]: done

File: common/djangoapps/third_party_auth/settings.py (hunk 2 above)
Comment (danialmalik) [reply common/djangoapps/third_party_auth/settings.py]: done

**Comment by asamolion** [comment]
Has this been manually tested?

**Comment by danialmalik** [comment]
I have tested and verified that required settings are loaded correctly. But successful event generation cannot be tested on our local devstack as google signin is required to test that.

**Comment by asamolion** [comment]
alright.

File: openedx/features/caliper_tracking/third_party/auth/constants.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+"""
```
Comment (asamolion) [inline openedx/features/caliper_tracking/third_party/auth/constants.py]: The entire `third_party` folder is unnecessary. You can simply add all of this into the `apps.py` file.

Comment (danialmalik) [reply openedx/features/caliper_tracking/third_party/auth/constants.py]: done

File: openedx/features/caliper_tracking/apps.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+
+from django.apps import AppConfig
+from django.conf import settings
+
+
+LOGIN_EVENT_UTIL = 'caliper_tracking.helpers.emit_login_event'
```
Comment (asamolion) [inline openedx/features/caliper_tracking/apps.py]: Please rename this to a more suitable identifier.

Comment (danialmalik) [reply openedx/features/caliper_tracking/apps.py]: done. please review now.

**Comment by asamolion** [comment]
Please squash and rebase before merging.

**Pull request merged**